.github/workflows/ci_e2e-mysql.yaml

```
@@ -205,11 +216,306 @@ jobs:
+                    grep -E '(pm\.|rlimit_core)' "$SYMFONY_FPM_INI" || echo "  (no pool or rlimit settings)"
+
+                    # Check if we need to modify config
+                    NEEDS_RESTART=false
+
+                    # Add pool configuration if not present or incorrect
+                    if ! grep -q "pm.max_children = 50" "$SYMFONY_FPM_INI"; then
+                        echo ""
+                        echo "Adding PHP-FPM pool configuration..."
+                        cat >> "$SYMFONY_FPM_INI" << 'FPMCONF'
+
+; Custom pool configuration for e2e tests (fixes 502 errors)
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 244: could not find expected ':'

(syntax-check)